src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -139,6 +127,45 @@ public void Initalize(IConfiguration configuration)
             }
         }
 
+        private void AddDefaultClrMetadataReferences(ProjectContext projectContext, HashSet<MetadataReference> commonReferences)
+        {
+            if (projectContext == null || projectContext.TargetFramework?.Framework != ".NETCoreApp")
+            {
+                var assemblies = new[]
+                {
+                    typeof(object).GetTypeInfo().Assembly,
+                    typeof(Enumerable).GetTypeInfo().Assembly,
+                    typeof(Stack<>).GetTypeInfo().Assembly,
+                    typeof(Lazy<,>).GetTypeInfo().Assembly,
+                    FromName("System.Runtime"),
+                    FromName("mscorlib")
+                };
+
+                var references = assemblies
+                    .Where(a => a != null)
+                    .Select(a => a.Location)
+                    .Distinct()
+                    .Select(l => _metadataFileReferenceCache.GetMetadataReference(l));
+
+                foreach (var reference in references)
+                {
+                    commonReferences.Add(reference);
+                }
+
+                Assembly FromName(string assemblyName)
```

Comment:
and here I was, so happy that I used a local function... 😋
you are right though

I also noticed it needs a rebase